deploy/olm-catalog/apicast-operator/apicast-operator.package.yaml

```diff
@@ -1,4 +0,0 @@
-packageName: apicast-operator-master
```

Comment:
No need to rename `apicast-operator/apicast-operator.package.yaml` to `apicast-community-operator/apicast-community-operator.package.yaml `, just the package name is enough. The less changes, the better.

Reply:
:+1: